### PR TITLE
Instruction 1.5

### PR DESCRIPTION
**Pull request by afestein**
Prints out the number of tweets collected for each keyword.

### CODE REVIEW

**Comment by afestein** [comment]
Something seems off with the output of this, because I get:
```
500
590
598
```

I thought the number of tweets retrieved was supposed to be limited to 500, so shouldn't we get 500 for each keyword?

**Comment by jjjjesse** [comment]
Apparently its a minimum of 500. I think the code just makes sure it's > 500

**Comment by afestein** [comment]
OK that seems fine then.

Gosh there's a lot of noise in these git diffs, I only changed a few lines 😬

File: Assignment2_notebook.ipynb [hunk 1]
```diff
@@ -1606,7 +1619,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.5.3"
+   "version": "3.6.1"
```
Comment (afestein) [inline Assignment2_notebook.ipynb:1622]: Hoping this does not cause any issues between our environments.

**Comment by jjjjesse** [comment]
yeah I know, massive pain

**Review by jjjjesse — APPROVED** [review]

**Pull request merged**